src/BizHawk.Client.Common/config/Config.cs

```diff
@@ -325,7 +322,6 @@ public void ResolveDefaults()
 			[VSystemID.Raw.GB] = CoreNames.Gambatte,
 			[VSystemID.Raw.GBC] = CoreNames.Gambatte,
 			[VSystemID.Raw.GBL] = CoreNames.GambatteLink,
-			[VSystemID.Raw.SGB] = CoreNames.Gambatte,

```

Comment:
This is incorrect, all `AppliesTo` sysIDs are used here (note GB and GBC). Actually I don't think the above change to `CorePickerUIData` is correct either.

Reply:
This has been restored in a commit. As for the other changes in the file, I have repeatedly switched between cores and it seems to work fine aside from the minor issue mentioned in the comment.